src/client/renderer/shaders/default_fs.glsl

```diff
@@ -74,6 +74,60 @@ vec3 eyeDir;
 
 out vec4 fragColor;
 
+/**
+ * @brief Cubic interpolation helper for textureBicubic().
+ */
+vec4 cubic(float v) {
+	vec4 n = vec4(1.0, 2.0, 3.0, 4.0) - v;
+	vec4 s = n * n * n;
+	float x = s.x;
+	float y = s.y - 4.0 * s.x;
+	float z = s.z - 4.0 * s.y + 6.0 * s.x;
+	float w = 6.0 - x - y - z;
+	return vec4(x, y, z, w) * (1.0/6.0);
```

Comment:
Just watch your whitespace in here, please :)

Reply:
Which parts? (I just totally copy/pasted this code from that source noted in the function below, btw)

Reply:
Just `1.0/6.0` should be `1.0 / 6.0` -- nit-picky, I know. I just like to maintain consistent whitespace for the entire codebase.